docs/admin/releases/06/README.rst

```diff
@@ -0,0 +1,132 @@
+
+.. contents:: :local:
+
+Changelog
+=========
+
+Improvements to scheduling checks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The "Scheduling Diagnostics" module has undergone significant improvements.
+Now, instead of waiting a long time for all the checks to be run and rendered,
+the page loads instantaneously, and you can run and see the results of
+whichever individual checks you want.
+
+If the "Scheduling Diagnostics" module is enabled for the program, you can get
+to the scheduling checks page by clicking the link "Run Scheduling Diagnostics"
+(/manage/[program]/[instance]/scheduling_checks) on the main program
+management.
+
+Class flags and class search improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The search interface formerly associated with class flags has been rewritten,
+and comes with new search options.  It will also now be easier to add more
+search options, so let us know if there are ways you would find it useful to be
+able to search. 
+
+Lottery Student Registration Big Board
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There is a new page for watching the current number of student registrations.
+You can get to it from the link "Student Registration Big Board" on the main
+program management page, or at /manage/[program]/[instance]/bigboard.  It has
+some of the same statistics as the dashboard, but is a lot faster to load, and
+has some fun extra numbers too.
+
+Logging current grade during student grade change request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Changes to the handling of student grade levels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tags to disable shirt types question in profile
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The newly added ``studentinfo_shirt_type_selection`` Tag allows you to disable
+the student profile question that asks about shirt types (plain vs fitted),
+while retaining the question that asks about shirt size. This complements the
+existing ``teacherinfo_shirt_type_selection`` Tag, which controls the same
+behavior for the teacher and volunteer profiles. For either of these Tags, set
+the value to False to disable the question.
+
+On the student profile, neither of the shirt questions appear unless the
+``show_student_tshirt_size_options`` Tag is set to True. On the teacher
+profile, both shirt questions appear by default, but can be simultaneously
+disabled by setting the ``teacherinfo_shirt_options`` Tag to False.
+
+Tags can be added/modified/removed from the /admin/tagdict/tag/ page of the
+admin panel.
+
+Customizing PDF letterhead
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Chapters can now upload a PDF letterhead file, which is used for the student
+completion certificate printable.  It should be named "letterhead.pdf" and go
+in the folder "latex_media" on the admin panel's file browser.  It can be a
+full-page PDF (8.5"×11"), or it can just be a top banner.
+
+UI for flushing individual caches
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
It seems unlikely that anyone will be able to use this without very specific instructions from a webmin -- for instance we might tell them "if the catalog class numbers are out of date, clear this cache" (as a workaround to #1463), but if it's just a general "something is out of date", they'll still want to flush the cache, since hunting down which cache to expire would really require a webmin.  For the former case, I'm not sure if it's worth having documentation so we can just say "go to this thing [link to docs] and clear `foo.bar.baz`", or if we'll have to explain more every time anyway and it's not worth trying.